packages-web/fieldset/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
The license is added in general for the whole mono repo in the root.

Reply:
@mijay can you give your contribution here.. should we have LICENSE for each widget or one for the whole mono repo?

Reply:
License on the top of the repository should be sufficient.

Reply:
Will remove it, we should think about including licenses inside release mpk's though.

Reply:
Removed it.